### ISSUE TITLE
[FEATURE]: Consider simplifying gaps configuration

### ISSUE THREAD

**Issue by gikari** [gikari/bismuth#104]
### Summary

In short, I want to unify outer gaps configurability: use only one spinbox instead of 4. I've never seen people using different values for Left, Right, Top and Bottom sides.

This would make adjusting the gaps much easier.

### A clear and concise description of what you want to happen

One spinbox to rule them all.

**Comment by gikari** [comment]
> >  I've never seen people using different values for Left, Right, Top and Bottom sides.
> 
> I have a super ultra wide monitor (5120x1440) and I set equal gaps on the right and left, and no gaps on the top and bottom.

Interesting. How does it look, could you give a screenshot?

**Comment by CaptainSpof** [comment]
Hi, I'd like to chime in, in favor of keeping the option as is (and possibly improve it).

I used to run to following setup:

![bismuth-gaps](https://user-images.githubusercontent.com/55411256/137750884-5fff11c2-0883-488c-b18f-77a01d1bd0e7.png)


where the multiple gaps allowed me to accommodate my top and bottom docks. I've since then stop doing that, because I've moved to a multi-screen setup, where the gaps are applied to each screen, which is not what I wanted, since I only have the dock in my main display. Leaving me with wasted space for the screen that don't have dock.

So ideally, I'd like to keep the flexibility and I'd love to gain the ability to apply those settings per monitor.

Thanks for your fork, it is much appreciated. :+1:
